semantic/src/site/globals/site.overrides

```diff
@@ -737,6 +737,136 @@ a:hover {
   margin-top: 25px;
 }
 
+/*--------------------------------
+      Dashboard
+---------------------------------*/
+
+// Layouts
+#dashboard {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+#dashboard .template-category {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+}
+
+#dashboard .template-category .title {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+
+#dashboard .templates {
```

Comment:
There's a lot of repetition here with existing flex layouts. It's fine for now, but at some point it'd be good to go through and consolidate flex layouts into a small set of re-useable classes.

Reply:
hmm, sure, i still not sure what the best practice is but defo can reduce repetion. just had a quick look, in less we can do this

how does it look after this commit 063886fe061b5e804983f9debde2ff7effa945e8 (hard to see the change in commit, best to look at overall diff
(added nesting, added .class(); within a class